### ISSUE TITLE
Gated attention issue in the paper

### ISSUE THREAD

**Issue by LightingMc** [ibm-granite/granite-tsfm#232]
There seems to be a small error in the paper. The authors claim that there is a dot product in gated attention. However, it is just implemented as a simple element-wise multiplication in the code. It is likely a minor typo and can be ignored, but I am wrong, is there a particular reason you call it a dot product?

code @ https://github.com/ibm-granite/granite-tsfm/blob/main/tsfm_public/models/tinytimemixer/modeling_tinytimemixer.py

class TinyTimeMixerGatedAttention(nn.Module):
    def __init__(self, in_size: int, out_size: int):
        super().__init__()
        self.attn_layer = nn.Linear(in_size, out_size)
        self.attn_softmax = nn.Softmax(dim=-1)

    def forward(self, inputs):
        attn_weight = self.attn_softmax(self.attn_layer(inputs))
        inputs = inputs * attn_weight
        return inputs

paper @ TSMIXER:

![image](https://github.com/user-attachments/assets/98a3ea58-5e6c-4eb5-81d5-578750257360)

**Comment by vijaye12** [comment]
Yes. Its a typo.. its a simple element-wise multiplication. Thanks for pointing the error in the appendix figure.. We will correct it soon.
